specification/alertsmanagement/resource-manager/readme.md

```diff
@@ -50,6 +50,15 @@ tag: package-2023-03
 
 
 
+
+### Tag: package-preview-2023-08
+
+These settings apply only when `--tag=package-preview-2023-08` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2023-08'
+input-file:
+  - Microsoft.AlertsManagement/preview/2023-08-01-preview/AlertRuleRecommendations.json
```

Comment:
It is recommended to copy all the APIs in new API Version, that's why errors in Avocado and Linter validations are appearing.

Reply:
Thanks, update the file and the errors are now gone